DESCRIPTION

```diff
@@ -21,7 +21,7 @@ Imports:
     methods,
     evaluate (>= 0.5.4),
     IRdisplay,
-    rzmq (>= 0.7.0),
+    pbdZMQ (>= 0.2.1),
```

Comment:
yep, not on CRAN. no matter if it'll exist or the next version will be 0.3.0, this will work :smiley: